ci-jobs/crowdin.yml

```diff
@@ -10,4 +10,6 @@ jobs:
         versionSpec: '11.x'
     - script: npm install
       displayName: NPM install
-    - script: npm run crowdin-update
\ No newline at end of file
+    - script: npm run crowdin-update
```

Comment:
it automatically picks up these vars from the Travis settings and it worked before. Why do we need to change it?

Reply:
This is azure pipelines. Secret variables need to be placed inside `env:`

Reply:
this script actually requires both project name and key variables to be defined

Reply:
also, it is only expected to be executed on merge to master and not for each commit

Reply:
I have it set to only trigger on master commits and the project name gets passed into env because it isnt a secret variable. AP has a rule about requiring explicitly passing secret variables to individual tasks.

Reply:
*is a secret variable